website/static/js/contribManager.js

```diff
@@ -391,6 +396,33 @@ var ContributorsViewModel = function(contributors, adminContributors, user, isRe
         });
     };
 
+    //TODO: investigate changing how sortable is added
+    self.afterRender = function(elements) {
+        rt.responsiveTable(elements[0].parentElement.parentElement);
```

Comment:
This function looks very brittle. 

- Dependence on order of HTML tags (`parentElement.parentElement`)
- Dependence on HTML tag types (`tbody`)
- Dependence on CSS IDs (`contributors`) and classes (`contrib-placeholder`)

Is there are reason we can't use knockout-sortable here?